src/_guides/language/language-tour.md

```diff
@@ -1588,6 +1587,11 @@ You can override many of these operators, as described in
   Operator precedence is an approximation of the behavior of a Dart parser.
   For definitive answers, consult the grammar in the
   [Dart language specification][].
+  {%- comment %}
+  Internal note from https://github.com/dart-lang/site-www/pull/2691#discussion_r506184100:
+  -  `??`, `&&` and `||` are excluded because they are lazy / short-circuiting operators
+  - `!` is probably excluded for historical reasons
+  {% endcomment %}
```

Comment:
Oh my! Apologies for my multitasking blunder. Done.

Reply:
Actually, just to be sure (since 3066 falls inside a code block for me), I've moved it just after the Note that starts with "You may have noticed that some [operators](#operators), like `!=`, are not in the list of names."